### ISSUE TITLE
[bug] can't compile tauri for development

### ISSUE THREAD

**Issue by SamuNatsu** [tauri-apps/tauri#10172]
### Describe the bug

cannot find type `Command` in module `crate::api::process`

failed to resolve: could not find `Encoding` in `process`

problem occurred on latest Tauri v1.7.0 while I'm creating a new tauri project on Win11 using command `pnpm create tauri-app`

same problem as #9728, and I've tried to update cargo dependencies, but nothing helpful

### Full `tauri info` output

```text
[✔] Environment
    - OS: Windows 10.0.22631 X64
    ✔ WebView2: 126.0.2592.81
    ✔ MSVC: Visual Studio Build Tools 2022
    ✔ rustc: 1.79.0 (129f3b996 2024-06-10)
    ✔ cargo: 1.79.0 (ffa9cf99a 2024-06-03)
    ✔ rustup: 1.27.1 (54dd3d00f 2024-04-24)
    ✔ Rust toolchain: stable-x86_64-pc-windows-msvc (default)
    - node: 20.15.0
    - pnpm: 9.5.0-beta.0
    - npm: 10.8.1

[-] Packages
    - tauri [RUST]: 1.7.0
    - tauri-build [RUST]: 1.5.3
    - wry [RUST]: 0.24.10
    - tao [RUST]: 0.16.9
    - @tauri-apps/api [NPM]: 1.6.0
    - @tauri-apps/cli [NPM]: 1.6.0

[-] App
    - build-type: bundle
    - CSP: unset
    - distDir: ../dist
    - devPath: http://localhost:1420/
    - framework: Vue.js
    - bundler: Vite
```

### Stack trace

```text
Compiling tauri v1.7.0
error[E0412]: cannot find type `Command` in module `crate::api::process`
   --> C:\Users\*****\scoop\persist\rustup\.cargo\registry\src\index.crates.io-6f17d22bba15001f\tauri-1.7.0\src\endpoints\shell.rs:244:41
    |
244 | ) -> super::Result<crate::api::process::Command> {
    |                                         ^^^^^^^ not found in `crate::api::process`
    |
help: consider importing this struct
    |
7   + use std::process::Command;
    |
help: if you import `Command`, refer to it directly
    |
244 - ) -> super::Result<crate::api::process::Command> {
244 + ) -> super::Result<Command> {
    |

error[E0433]: failed to resolve: could not find `Encoding` in `process`
   --> C:\Users\*****\scoop\persist\rustup\.cargo\registry\src\index.crates.io-6f17d22bba15001f\tauri-1.7.0\src\endpoints\shell.rs:304:50
    |
304 |     if let Some(encoding) = crate::api::process::Encoding::for_label(encoding.as_bytes()) {
    |                                                  ^^^^^^^^ could not find `Encoding` in `process`
    |
help: consider importing this struct
    |
7   + use encoding_rs::Encoding;
    |
help: if you import `Encoding`, refer to it directly
    |
304 -     if let Some(encoding) = crate::api::process::Encoding::for_label(encoding.as_bytes()) {
304 +     if let Some(encoding) = Encoding::for_label(encoding.as_bytes()) {
    |

   Compiling vswhom v0.1.0
   Compiling embed-resource v2.4.2
   Compiling tauri-winres v0.1.1
   Compiling tauri-build v1.5.3
Some errors have detailed explanations: E0412, E0433.
For more information about an error, try `rustc --explain E0412`.
error: could not compile `tauri` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
```

**Comment by yuqiang-yuan** [comment]
The same. And my `tauri info` is: 

```
yarn tauri info                                                                                                                     ✔  base   17:08:04 
yarn run v1.22.19
$ tauri info

[✔] Environment
    - OS: Mac OS 14.5.0 X64
    ✔ Xcode Command Line Tools: installed
    ✔ rustc: 1.79.0 (129f3b996 2024-06-10)
    ✔ cargo: 1.79.0 (ffa9cf99a 2024-06-03)
    ✔ rustup: 1.27.1 (54dd3d00f 2024-04-24)
    ✔ Rust toolchain: stable-aarch64-apple-darwin (default)
    - node: 18.19.0
    - pnpm: 7.26.2
    - yarn: 1.22.19
    - npm: 10.2.3

[-] Packages
    - tauri [RUST]: 1.7.0
    - tauri-build [RUST]: 1.5.3
    - wry [RUST]: 0.24.10
    - tao [RUST]: 0.16.9
    - tauri-cli [RUST]: 1.5.11
    - @tauri-apps/api [NPM]: 1.6.0
    - @tauri-apps/cli [NPM]: 1.6.0

[-] App
    - build-type: bundle
    - CSP: unset
    - distDir: ../dist
    - devPath: http://localhost:1420/
    - framework: Vue.js
    - bundler: Vite
✨  Done in 7.17s.
```

**Comment by FabianLars** [comment]
[pr](https://github.com/tauri-apps/tauri/pull/10174) is up, thanks for the report.

**Comment by yq612** [comment]
same ques

**Comment by gabrielalexandrelopes** [comment]
Is there a way to revert to the previous version of tauri? I tried changing the version in the Cargo.toml but no success.

**Comment by doums** [comment]
@gabrielalexandrelopes you have to reset Cargo.lock as well

**Comment by gabrielalexandrelopes** [comment]
@doums  I started from scratch:
```
cargo install create-tauri-app --locked --force
cargo create-tauri-app
cd tauri-app
cargo tauri dev
```

but I'm still getting the same error. How can I reset? rm the lock file? (sorry I'm a beginner in rust)

**Comment by FabianLars** [comment]
Alternatively just wait 10-20 minutes for CI to be done releasing the fix :)

Then you can run `cargo update` in your src-tauri dir and it should pull in the fix (or create a new project)

**Comment by gabrielalexandrelopes** [comment]
Sounds good, thanks @FabianLars !

**Comment by SamuNatsu** [comment]
problem solved on tauri v1.7.1

thx
